doc/source/development/extending.rst

```diff
@@ -208,6 +208,194 @@ will
 2. call ``result = op(values, ExtensionArray)``
 3. re-box the result in a ``Series``
 
+:class:`~pandas.api.extensions.ExtensionArray` Series Operations Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 0.25.0
+
+In addition to operators like `__mul__` and `__add__`, the pandas Series
+namespace provides a long list of useful operations such as :meth:`Series.round`,
+:meth:`Series.sum`, :meth:`Series.abs`, etc'. Some of these are handled by
+pandas own algorithm implementations (via a dispatch function), while others
+simply call an equivalent numpy function with data from the underlying array.
+In order to support this operations in a new ExtensionArray, you must provide
+an implementation for them.
+
+.. note::
+
+    There is a third category of operation which live on the `pandas`
+    namespace, for example `:meth:pd.concat`. There is an equivalent
+    numpy function `:meth:np.concatenate`, but this is not used. n
+    general, these function should just work with your EA, you do not
+    need to impelment more than the general EA interface.
+
+As of 0.25.0, the list of series operations which pandas' provides its own
```

Comment:
capital S for Series, and please rephrase

Reply:
Fixed.